.github/actions/setup-next/action.yml

```diff
@@ -0,0 +1,14 @@
+name: "Setup Next.js"
+description: "Installs node dependencies, with cache, and gets Next up and running, also with a cache."
```

Comment:
Nit: this only does the nextjs setup + cache since the node stuff is in the other action

Reply:
Yep, thanks, originally this did both and I forgot to change the description.